x-pack/plugins/security_solution/public/timelines/components/timeline/header/title_and_description.tsx

```diff
@@ -67,12 +70,21 @@ const usePrevious = (value: unknown) => {
 // the modal is used as a reminder for users to save / discard
 // the unsaved timeline / template
 export const TimelineTitleAndDescription = React.memo<TimelineTitleAndDescriptionProps>(
-  ({ closeSaveTimeline, initialFocus, openSaveTimeline, timelineId, showWarning }) => {
+  ({
+    closeSaveTimeline,
+    currentDescription,
+    currentTitle,
+    initialFocus,
+    timelineId,
+    showWarning,
+    updateCurrentTitle,
+    updateCurrentDescription,
+  }) => {
```

Comment:
can we please refactor that to `useForm` as it seems it will be a way cleaner and we don't need to open follow-up PR shortly after?